src/components/index.js

```diff
@@ -27,7 +26,12 @@ import WalletViewTitle from './WalletViewTitle'
 import NewWalletButtons from './NewWalletButtons'
 import ButtonWithConfirm from './ButtonWithConfirm'
 import MnemonicAddresses from './MnemonicAddresses'
+import DigitalAssetsGrid, { type DigitalAssetsGridItemType } from './DigitalAssetsGrid'
```

Comment:
please let's avoid exporting of components types from this file
it is already quite complex without it

Reply:
Will be fixed in the next PR with digital-assets manager